src/world/Game/Storage/ObjectStorage.cpp

```diff
@@ -308,7 +308,8 @@ void ObjectMgr::LoadExtraCreatureProtoStuff()
 				{
 					cn->m_canFlee = true;
 					if(sp->floatMisc1)
-						cn->m_canFlee = (sp->floatMisc1 > 0.0f ? true : false);
```

Comment:
Are we sure all creatures (that includes bosses, guards, etc too) will flee?

Reply:
uhm, i checked a couple of creatures wich dont have agent flee entry in the table and they wasnt fleeing... im gonna check again tomorrow.

Reply:
You want to have some check to prevent in case of someone add a flee agent for a boss or guard?

Reply:
No I don't think it should be fool-proof, maybe I just misunderstood something. I will check again :)

Reply:
OK, I get it now, I am just blind sometimes :D
I didn't notice that this code was only executing for the FLEE agent.
Sorry about that!